.github/workflows/build_make.yml

```diff
@@ -0,0 +1,210 @@
+        with:
+          path: ~/.opam
+          key: v1-${{ runner.os }}-alt-ergo-${{ matrix.cache-name }}-${{ env.OCAML_DEFAULT_VERSION }}-${{ hashFiles('*.opam') }}
+
+      - name: Use OCaml ${{ env.OCAML_DEFAULT_VERSION }}
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: ${{ env.OCAML_DEFAULT_VERSION }}
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: Pin no action
+        run: opam pin add . -y --no-action
+
+      - name: Alt-Ergo-lib
+        run: |
+          opam depext -y alt-ergo-lib
+          opam install -y ./alt-ergo-lib.opam --deps-only --with-test
+          opam exec -- ./configure alt-ergo-lib
+          opam exec -- make alt-ergo-lib
+
+      - name: Alt-Ergo-parsers
+        run: |
+          opam depext -y alt-ergo-parsers
+          opam install -y ./alt-ergo-parsers.opam --deps-only --with-test
+          opam exec -- ./configure alt-ergo-parsers
+          opam exec -- make alt-ergo-parsers
+
+      - name: Alt-Ergo
+        run: |
+          opam depext -y alt-ergo
+          opam install -y ./alt-ergo.opam --deps-only --with-test
+          opam exec -- ./configure alt-ergo
+          opam exec -- make alt-ergo
+
+      - name: Altgr-Ergo
+        run: |
+          opam depext -y altgr-ergo
+          opam install -y ./altgr-ergo.opam --deps-only --with-test
+          opam exec -- ./configure altgr-ergo
+          opam exec -- make altgr-ergo
+
+  make_install:
+    # Test every install/uninstall rules
+    name: Make install
+
+    # Only if in pull request
+    if: github.event_name == 'pull_request'
+
+    needs: make_all
+
+    strategy:
+      matrix:
+        cache-name:
+          - make-install
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Retrieve opam cache
+        uses: actions/cache@v2
+        id: cache-opam
+        with:
+          path: ~/.opam
+          key: v1-${{ runner.os }}-alt-ergo-${{ matrix.cache-name }}-${{ env.OCAML_DEFAULT_VERSION }}-${{ hashFiles('*.opam') }}
+
+      - name: Use OCaml ${{ env.OCAML_DEFAULT_VERSION }}
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: ${{ env.OCAML_DEFAULT_VERSION }}
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: Pin no action
+        run: opam pin add . -y --no-action
+
+      - name: Build
+        run: |
+          opam depext -y alt-ergo-lib alt-ergo-parsers alt-ergo altgr-ergo
+          opam install -y ./*.opam --deps-only --with-test
+          opam exec -- make
+
+      - name: Install/Uninstall Alt-Ergo-lib
+        run: |
+          opam exec -- make install-lib
+          opam exec -- make uninstall-lib
```

Comment:
If you really want to test the installation/uninstallation process, then you should:
- after installing, use the installed version (there was an old script doing that iirc) and/or check the existence of files at the installation destination
- after uninstalling, ensure that no leftover files are left

This also applied to the the parser lib, the binary (which is easier to test), and the gui (more tricky to test)

Reply:
I've added a script that test if the bins (bin and gui install rules) and the libs (lib and parsers install rules) are accessible either with ocamlfind for a lib or by running `<bin> --version`.
This command must succed after the `install` step and must fails when we run the script after the `uninstall` step.